### ISSUE TITLE
Support for alignment and standoff annotation

### ISSUE THREAD

**Issue by jnivre** [UniversalDependencies/UD_v2#25]
The v2 proposal should also contain instructions on how to add (a) standoff annotation on top of treebanks, (b) sentence and word alignments for parallel treebanks. I believe some of our colleagues in Prague already have a proposal for (b).

**Comment by dan-zeman** [comment]
@martinpopel has a proposal for (b) and I think he has even sketched it in https://github.com/UniversalDependencies/docs/issues/321

But he wanted to use the `DEPS` column for that and I'm afraid our current v2 proposal does not enable what he had in mind.

**Comment by martinpopel** [comment]
Yes, in UniversalDependencies/docs#321, I proposed multiple _zones_ in one CoNLL-U file and sent_id. Both is needed for storing alignment (and standoff annotation and several other useful things). Once we agree on that issue (so far it has been approved by @dan-zeman and @manning), we can proceed to the exact way of storing alignment.

My proposal is to store alignment as directed links, so it can have the same low-level API as coreference and secondary dependencies. We could store them in MISC, but I would prefer to store it in the column which is now called DEPS and could be renamed e.g. to LINKS (or RELATIONS). It would be a list of pairs: node-id and type_of_relation. There could be shortcuts for UDv1 style of secondary dependencies where the node-id misses the bundle and zone identification. See a [sample](http://ufallab.ms.mff.cuni.cz/~popel/czeng1.6-sample.conllu). I need to study what are the changes in the v2 proposal and why they may not enable this.

**Comment by jnivre** [comment]
I am sorry for not devoting enough attention to #321 (until now), but it would be really good to investigate its compatibility with the v2 proposal.
